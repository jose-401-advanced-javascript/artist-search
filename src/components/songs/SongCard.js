import React from 'react';
import PropTypes from 'prop-types';
// import styles from './AlbumCard.css';
import { Link } from 'react-router-dom';

const SongCard = ({ id, title, name }) => {

  return (
    <div key={id}>
      <Link to={`/lyrics/${name}/${title}`}>
        <p>{title}</p>
      </Link>
    </div>
  );
};

SongCard.propTypes = {
  id: PropTypes.string.isRequired,
  title: PropTypes.string.isRequired,
  name: PropTypes.string.isRequired
};

export default SongCard;