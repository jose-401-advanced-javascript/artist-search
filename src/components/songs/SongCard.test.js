import React from 'react';
import { shallow } from 'enzyme';
import SongCard from './SongCard';

describe('SongCard component', () => {
  it('renders SongCard', () => {
    const wrapper = shallow(<SongCard id={'asdgadg'} title={'adsgadsg'} name={'asdg'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
