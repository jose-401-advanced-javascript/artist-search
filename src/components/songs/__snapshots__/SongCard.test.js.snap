// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SongCard component renders SongCard 1`] = `
<div
  key="asdgadg"
>
  <Link
    to="/lyrics/asdg/adsgadsg"
  >
    <p>
      adsgadsg
    </p>
  </Link>
</div>
`;
